smoke/tests/native_layer_test.go

```diff
@@ -156,6 +157,65 @@ func (n *NativeLayerTestSuite) testMakeLayers(ctx tool.Context, t *testing.T) {
 	tool.Verify(t, ctx, lowerLayer.FileTree)
 }
 
+// specify blob dir and cache dir with option `localfs-dir`
+func (n *NativeLayerTestSuite) TestSharedCacheDirWithBlobDir(t *testing.T) {
```

Comment:
Why do we need to add an independent case for `local-fs-dir`, can we reuse `TestMakeLayers` ?

Reply:
> Why do we need to add an independent case for `local-fs-dir`, can we reuse `TestMakeLayers` ?

Because the workflow is not same, `testMakeLayers` does not support this option.

Reply:
Can we make it reusable?